manpages/fsck.fat.8.in

```diff
@@ -19,82 +19,82 @@
 .\" can be found in /usr/share/common-licenses/GPL-3 file.
 .\"
 .\"
-.TH FSCK.FAT 8 2017\-06\-28 "dosfstools @PACKAGE_VERSION@"
+.TH FSCK.FAT 8 2017-06-28 "dosfstools @PACKAGE_VERSION@"
 .SH NAME
-fsck.fat \- check and repair MS\-DOS FAT filesystems
+fsck.fat \- check and repair MS-DOS FAT filesystems
 .\" ----------------------------------------------------------------------------
 .SH SYNOPSIS
-\fBfsck.fat\fR [\fIOPTIONS\fR] \fIDEVICE\fR
+\fBfsck.fat\fP [\fIOPTIONS\fP] \fIDEVICE\fP
```

Comment:
Same rewrite needed as for fatlabel(8); not mentioning that again in following files.